microsoft-edge/devtools-guide-chromium/css/css-in-js.md

```diff
@@ -50,7 +50,7 @@ This feature is available starting with Microsoft Edge version 93. <!-- delete s
 
 <!-- from https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools#style-editing-for-css-in-js-frameworks -->
 
-The **Styles** pane supports editing styles that were created with the [CSS Object Model (CSSOM)](https://drafts.csswg.org/cssom) APIs.  Many CSS-in-JS frameworks and libraries use the CSS Object Model APIs under the hood to construct styles.
+The **Styles** pane supports editing styles that were created with the [CSS Object Model (CSSOM)](https://drafts.csswg.org/cssom/) APIs.  Many CSS-in-JS frameworks and libraries use the CSS Object Model APIs under the hood to construct styles.
```

Comment:
I don't see which error # but get "ERR_NAME_NOT_RESOLVED":
https://drafts.csswg.org/cssom/
Hmmm… can't reach this pagedrafts.csswg.org’s server IP address could not be found.
Try:
Checking the connection
Checking the proxy, firewall, and DNS settings.
Running Windows Network Diagnostics
ERR_NAME_NOT_RESOLVED

Reply:
entered in your fix